tests/test_oml/test_functional/test_metrics/test_topological_metrics.py

```diff
@@ -0,0 +1,59 @@
+from typing import Callable, Tuple
```

Comment:
After you have PCA as a separate function, you can test that it outputs the same as sklearn implementation. For this, I suggest you create a matrix of some values and copy the answer from sklearn here in order to use it as expected values.

To construct the matrix I suggest using `arrange()` in order not to deal with random and oversimplified examples constructed via `zeros` and `ones`. Please, check the alignment of intermediate values as well (not only the resulting matrix but explained variance as well).